### PR TITLE
Update gnupg, add USE=usb dependencies

### PR DESCRIPTION
**Pull request by euank**
This updates gnupg and adds the dependencies needed to build it with usb smartcard support.

This is useful for our sdk where developer's may with to run gpg-agent + interact with usb smartcards.

### CODE REVIEW

File: dev-libs/libusb/libusb-1.0.21.ebuild [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+EAPI=5
+inherit eutils toolchain-funcs multilib-minimal
+
+DESCRIPTION="Userspace access to USB devices"
+HOMEPAGE="http://libusb.info/ https://github.com/libusb/libusb"
+SRC_URI="https://github.com/${PN}/${PN}/releases/download/v${PV}/${P}.tar.bz2"
+
+LICENSE="LGPL-2.1"
+SLOT="1"
+KEYWORDS="alpha amd64 arm ~arm64 ~hppa ia64 ~m68k ~mips ppc ppc64 ~s390 ~sh sparc x86 -x86-fbsd ~amd64-linux ~arm-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos"
```
Comment (dm0-) [inline dev-libs/libusb/libusb-1.0.21.ebuild]: Accept `~arm64` in the overlay PR?

File: virtual/libusb/libusb-1-r2.ebuild [hunk 2]
```diff
@@ -1,17 +1,12 @@
...
 inherit multilib-build
 
 DESCRIPTION="Virtual for libusb"
-HOMEPAGE=""
-SRC_URI=""
-
-LICENSE=""
 SLOT="1"
-KEYWORDS="~alpha amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc x86 ~amd64-fbsd ~x86-fbsd ~x86-freebsd ~amd64-linux ~arm-linux ~ia64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos"
+KEYWORDS="~alpha amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~arm-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos"
```
Comment (dm0-) [inline virtual/libusb/libusb-1-r2.ebuild]: It probably doesn't matter as much for virtual packages, but this version is out of sync between amd64 and arm64.

Comment (euank) [reply virtual/libusb/libusb-1-r2.ebuild]: It matters even less since it's only pulled in for the sdk (USE=usb)

Comment (dm0-) [reply virtual/libusb/libusb-1-r2.ebuild]: Looks like it's used by usbutils in prod images, but yes, it's not an important difference.

**Pull request merged**